op-node/cmd/batch_decoder/main.go

```diff
@@ -55,6 +55,11 @@ func main() {
 					Usage:    "L1 RPC URL",
 					EnvVars:  []string{"L1_RPC"},
 				},
+				&cli.IntFlag{
+					Name:  "parallel",
```

Comment:
any better flag name?

Reply:
For [terraform](https://developer.hashicorp.com/terraform/cli/commands/apply#parallelism-n), [-parallelism=n](https://developer.hashicorp.com/terraform/cli/commands/apply#parallelism-n) - Limit the number of concurrent operation as Terraform [walks the graph](https://developer.hashicorp.com/terraform/internals/graph#walking-the-graph). Defaults to 10.

Chatgpt recommended `--concurrent-requests`, or `--num-concurrent-requests`.

How do you think? I think `--concurrent-requests` is verbose but fine.



Go test supports [-parallel](https://stackoverflow.com/questions/44325232/are-tests-executed-in-parallel-in-go-or-one-by-one/44326377#44326377) flag.

Reply:
Applied `concurrent-requests`.